server/chat-plugins/chat-monitor.js

```diff
@@ -139,6 +139,13 @@ Chat.registerMonitor('evasion', {
 	punishment: 'EVASION',
 	label: 'Filter Evasion Detection',
 	monitor(line, room, user, message, lcMessage, isStaff) {
+		// Many codepoints used in filter evasion detection can be decomposed
+		// into multiple codepoints that are canonically equivalent to the
+		// original. Perform a canonical composition on the message to detect
+		// when people attempt to evade by abusing this behaviour of Unicode.
+		message = message.normalize('NFKC');
```

Comment:
```suggestion
```
I would at least remove this one; it shouldn't be used for filtering in the first place.

Reply:
Yeah this doesnt seem necessary. Message in this function is only used to check if the exact word was used, as opposed to evading using something stripped in a message. I think this only harms that behavior.